### PR TITLE
Add support to host under a subpath like /random_psk_here/

### PR DESCRIPTION
**Pull request by ypid**
For background see https://github.com/debops/debops/issues/1233

### CODE REVIEW

File: etesync_server/wsgi.py [hunk 1]
```diff
@@ -10,7 +10,24 @@
 import os
 
 from django.core.wsgi import get_wsgi_application
+from django.conf import settings
```
Comment (tasn) [inline etesync_server/wsgi.py]: I guess this line is a mistake and should be removed?

Comment (ypid) [reply etesync_server/wsgi.py]: You are right. I removed it.

**Comment by tasn** [comment]
Ah sorry, after reading it further, it doesn't seem like this setting is even needed.
Take a look at http://albertoconnor.ca/hosting-django-under-different-locations.html it looks like it's supported out of the box.

**Comment by ypid** [comment]
Thanks, `SCRIPT_NAME` set by Nginx works nicely. Not sure how I could overlook that. I actually already set `X-Script-Name` before but it was wrong.

Closing and keep up the good work :)

**Comment by tasn** [comment]
Great to hear. :)

**Pull request closed without merging**